CMakeLists.txt

```diff
@@ -286,6 +286,8 @@ if(SYMFORCE_BUILD_CC_SYM)
   add_subdirectory(symforce/pybind)
 endif()
 
+add_subdirectory(symforce/slam)
```

Comment:
I'm fine conditioning it on something (I can imagine someone not wanting to bother building it if they aren't using it), but I'll just point out that the `symforce_slam` target depends only on `symforce_gen` and `Eigen`.

Reply:
Hmm, that's a good point - it does seem like you probably only need `symforce/slam` if you're building `symforce_opt`, but technically not.  It seems a little weird to just always build it like we do for symforce_gen, but I guess that's fine

Reply:
Okay, since we added the `ImuPreintegrator::Factor` method, I needed to import `sym::Factor` and `sym::Key`, so now `symforce_slam` does depend on `symforce_opt`, and I moved it to the `if(SYMFORCE_BUILD_OPT)` conditional.